server/routes/api/__test__/auth.unit.test.js

```diff
@@ -48,3 +48,34 @@ describe('POST Auth/Login', () => {
     expect(res.body.email).toEqual(email); // returned email should be correct user
   });
 });
+
+describe('POST Auth/Forgot', () => {
+  const existingUserEmail = 'testUser@test.com';
+
+  beforeAll(() => User.create({ email: existingUserEmail, password: 'test' }));
+
+  afterAll(() => User.findOneAndDelete({ email: existingUserEmail }));
+  it('will return status 200 on nonexistent user', async () => {
+    const res = await request(app)
+      .post('/api/auth/forgot')
+      .type('form')
+      .send({ email: 'doesnotexist@test.com' });
+    expect(res.statusCode).toEqual(200);
+  }); // send 200 on non existent user email instead of error for security purposes.
+  it('will return status 200 on existing user', async () => {
+    const res = await request(app)
+      .post('/api/auth/forgot')
+      .type('form')
+      .send({ email: existingUserEmail });
+    expect(res.statusCode).toEqual(200);
+  }); // send 200 on existent user email.
```

Comment:
This comment isn't needed. What's in the `it()` should describe this well enough.